WordPress-Core/ruleset.xml

```diff
@@ -150,9 +150,7 @@
 		<!-- Covers rule: When type casting, do it like so: $foo = (boolean) $bar; -->
 		<rule ref="Generic.Formatting.SpaceAfterCast"/>
 		<rule ref="WordPress.WhiteSpace.CastStructureSpacing"/>
-		<rule ref="WordPress.WhiteSpace.CastStructureSpacing.ContainsWhiteSpace">
-			<type>warning</type>
-		</rule>
+		<rule ref="WordPress.WhiteSpace.CastStructureSpacing.ContainsWhiteSpace"/>
```

Comment:
Line 152 above already includes the whole of CastStructureSpacing, so is this extra explicit line 153 really needed? I think probably not, now where not passing it any custom config.